02-peer-protocol.md

```diff
@@ -630,6 +634,12 @@ reply with the same value (completing after three messages).
     2. data:
         * [`u64`:`min_fee_satoshis`]
         * [`u64`:`max_fee_satoshis`]
+    1. type: 2 (`local_musig2_pubnonce`)
+    2. data:
+        * [`66*byte`:`nonces`]
+    1. type: 4 (`remote_musig2_pubnonce`)
```

Comment:
Why are two sets of nonces needed here? I was under the impression that in the co-op close case, there's actually just one message being signed, so we can get away with a single nonce?